suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx

```diff
@@ -24,6 +23,16 @@ type TransactionTypeInfo = {
     iconName?: IconName;
 };
 
+export const signValueMap = {
```

Comment:
Don't we already have this exact object somewhere else?

Reply:
We don't I moved it here from different file to avoid circular dependency.